lib/reporters/mini.js

```diff
@@ -10,6 +11,10 @@ function MiniReporter() {
 		return new MiniReporter();
 	}
 
+	var spinnerDef = spinners.dots;
+	this.spinnerFrames = spinnerDef.frames;
+	this.spinnerInterval = spinnerDef.interval;
```

Comment:
@sindresorhus usually objects to that. I don't have a strong opinion either way.